core/src/main/java/org/geysermc/geyser/GeyserImpl.java

```diff
@@ -621,12 +624,18 @@ public void reload() {
      *
      * @return true if the version number is not 'DEV'.
      */
+    @SuppressWarnings("BooleanMethodIsAlwaysInverted")
     public boolean isProductionEnvironment() {
         // First is if Blossom runs, second is if Blossom doesn't run
         //noinspection ConstantConditions,MismatchedStringCase - changes in production
         return !("git-local/dev-0000000".equals(GeyserImpl.GIT_VERSION) || "${gitVersion}".equals(GeyserImpl.GIT_VERSION));
     }
 
+    public boolean isDevBuild() {
+        //noinspection ConstantValue - changes in production
+        return !GeyserImpl.BRANCH.equals("master") && !GeyserImpl.REPOSITORY.equalsIgnoreCase("https://github.com/GeyserMC/Geyser");
+    }
```

Comment:
What does the repository change to?

Reply:
on a pr, the value is set to "DEV" by the build.gradle.kts file, and I'd assume it's the fork when it's someone's Geyser fork

Reply:
lets just instead declare that its a fork when a repo is different

master branch vs different branch is kinda meaningless if its a fork

Reply:
I would agree, but then Geyser PR builds would also be included.
You'd think that the branch would be the authors branch name - which it usually is - but in our case, for PRs, it falls back to "DEV". 

See `git.properties` file of a build of this pr:

```
git.branch=DEV
git.build.number=4031
git.build.version=2.2.0-SNAPSHOT
git.commit.id=9c69058d9b4b219d0cb5dc21ab79ea08917086f8
git.commit.id.abbrev=9c69058
git.commit.message.full=
git.remote.origin.url=https://github.com/GeyserMC/Geyser
```

Master (downloaded from the downloads site):

```
git.branch=master
git.build.number=393
git.build.version=2.2.0-SNAPSHOT
git.commit.id=3d05181a1390ff43d66ed8e8b79792bf883df859
git.commit.id.abbrev=3d05181
git.commit.message.full=
git.remote.origin.url=https://github.com/GeyserMC/Geyser
```

For builds from someone's fork, yes, i agree that master branch checking is meaningless, hence why the repo is checked.
Ideally, all builds that aren't from the GeyserMC/Geyser repo's master branch would be shown as dev builds (at least from my point of view).

Reply:
> I would agree, but then Geyser PR builds would also be included.
You'd think that the branch would be the authors branch name - which it usually is - but in our case, for PRs, it falls back to "DEV".

I was talking about the REPOSITORY field in GeyserImpl though? Sorry I'm just curious and possible confused lol

Reply:
The repository field changes when the build is on someone's fork.

There are basically these 3 scenarios:

A build is marked as dev, if:
- the branch name is "DEV" -> happens for Geyser PRs
- the repository isn't GeyserMC/Geyser -> happens for all Geyser forks (for my action runs, it's onebeastchris/Geyser)
- the branch name isn't "master" on "GeyserMC/Geyser" -> e.g. other geyser branches

Checking for only one thing would basically ensure that it's not that complete